topostats/tracing/nodestats.py

```diff
@@ -853,7 +853,7 @@ def analyse_node_branches(
         if len(crossing_quants) <= 1:
             conf = None
         else:
-            combs = nodeStats.get_two_combinations(crossing_quants)
+            combs = get_two_combinations(crossing_quants)
```

Comment:
Perhaps obvious but we should be explicit and generally try and avoid abbreviations (`combs` are also things we style our hair with!).

```suggestion
            combinations = get_two_combinations(crossing_quants)
```

Reply:
"combs" -> "combinations" and "comb" -> "pair" (when looping through "combinations") across all of the newly added files